### ISSUE TITLE
configuration should be more easily modified

### ISSUE THREAD

**Issue by rokroskar** [SwissDataScienceCenter/renku-jena#3]
Files like `shiro.ini` and `config.ttl` should be more easily overriden via the chart's values. They need to be templated so their specific values can be configurable via the values file.

**Comment by jachro** [comment]
Yes, both mentioned files are in the docker just for reference and to make Jena start when someone tries to play with it and have some reasonable defaults. README says they have to be mounted with all the instructions. Actually, the `config.ttl` as-is will probably be fine in the majority of use-cases in the set-up where the so-called, Fuseki Full is used.

**Comment by rokroskar** [comment]
Oh I see, via `additionalVolumeMounts` - I think it would still be nicer if we could have a dedicated value for the content of each of those configmaps to do this boilerplate. Then the content is also persisted in the values file instead of separately somewhere.

**Comment by jachro** [comment]
There's only one problem with that. We use the images in graph unit and acceptance tests which is quite handy that they are ready to work without many ceremonies. Personally, I'd say it's a good compromise.

**Comment by rokroskar** [comment]
You can keep the docker images as they are with the reasonable defaults - I'm just saying we can set up the chart to override them more easily.

**Comment by rokroskar** [comment]
I adjusted the wording at the top of the issue
